src/gui/PianoView.cpp

```diff
@@ -618,6 +619,9 @@ void PianoView::mouseMoveEvent( QMouseEvent * _me )
  */
 void PianoView::keyPressEvent( QKeyEvent * _ke )
 {
+#ifdef LMMS_BUILD_LINUX
```

Comment:
> Not exactly. It is automatically determined when assembling LMMS

I'm not sure your point here, yes I understand we can do this with `#ifdef`'s.  I'm asking we please don't.